src/client/src/apps/MainRoute/widgets/spotTile/components/analyticsTile/AnalyticsTile.tsx

```diff
@@ -73,18 +89,28 @@ class AnalyticsTile extends React.PureComponent<SpotTileProps> {
                 showResetButton={showResetButton}
                 disabled={inputDisabled}
               />
+              {showTimer && rfqTimeout !== null && rfqReceivedTime !== null && (
+                <RfqTimer
+                  onRejected={handleRfqRejected}
+                  receivedTime={rfqReceivedTime}
+                  timeout={rfqTimeout}
+                  isAnalyticsView={true}
```

Comment:
You can omit the `={true}` for React boolean props with a static value of `true`.
```suggestion
                  isAnalyticsView
```